src/main/steps/applicant1/interim-applications/dispense-service/da-search/content.ts

```diff
@@ -94,6 +96,14 @@ export const form: FormContent = {
           label: l => l.yes,
           id: 'yes',
           value: YesOrNo.YES,
+          subFields: {
+            conditionalControlHint: {
+              type: 'hidden',
+              label: l => l.finalOrderSearchHint,
+              labelHidden: true,
+              hint: l => l.finalOrderSearchHint,
+            },
+          },
```

Comment:
There are warning and conditionalText options for radios, that make a message show when the option is selected, e.g:
```suggestion
             conditionalText: l => l.finalOrderSearchHint,
          },
```